app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -482,6 +480,21 @@ class GithubWebhook extends RequestHandler<Body> {
     }
   }
 
+  bool _isReleaseBranch(PullRequest pr) {
+    final String defaultBranchName = Config.defaultBranch(pr.base!.repo!.slug());
+    final String baseName = pr.base!.ref!;
+
+    if (baseName == defaultBranchName) {
```

Comment:
This is enough. The connotation in infra is that a release branch is anything that isn't the default branch

Reply:
Thanks for the reply! umm this would short circuit the logic of `_getWrongBaseComment`. Previously branches of the format not conforming to flutter-release-x will receive `_getWrongBaseComment`. If we change the logic to only check for default branch alignment, then the block of code of `_getWrongBaseComment` should be removed?

Reply:
No, let's leave it in. Can you leave a comment explaining why we explicitly check the format?